### DISCUSSION TITLE
Camera follows an object.

### DISCUSSION THREAD

**Discussion by Throndax57** [AdamsLair/duality#885] [Questions & Help] [answered]
I'm not sure if this is the right place to ask a question but how can I make the Main Camera change its position relative to a position change on another object(lets say player).
I'm sorry if I'm asking a tedious question.

**Comment by SirePi** [comment, marked as answer]
There are no tedious questions :)

In order to do what you want, there are basically two ways:

1. Create a Component that implements `ICmpUpdatable` and in its `Update` method have it track your target GameObject's (player, enemy, whatever) Position, and attach it to the Camera.
2. Simply have the Camera as a children to the GameObject that you want to track.

If you need more info or it's not clear, just let me know

**Reply by Throndax57** [reply]
Edit: 
Thank you, I got it working now!!!
